tests/integration/resources/testpod.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: testpod
+spec:
+  containers:
+  - image: dataplatformoci/spark:3.3.2@sha256:5d5b02b7fcc60f86fa07c9aefbe98245d5030e10be12af9968b8b9b545a97355
```

Comment:
@delgod and @deusebio 

I had to explicitly add the latest SHA256 id here to work with the latest ROCK image to create the test pod.

Can you please share if this can be removed for CI where runners do not have any cached previous version?

Reply:
uhm, you mean that otherwise it does work locally or in CI? I would say that in CI, since there is no image downloaded yet, k8s would pull the latest image on dockerhub, which should already have the right hash.

Am I missing something?